templates/jenkins-s2i-build-template-with-secret.yml

```diff
@@ -0,0 +1,142 @@
+---
```

Comment:
Would you be open to add or merge this template to the existing [jenkins-s2i-build](https://github.com/redhat-cop/openshift-templates/tree/master/jenkins-s2i-build) templates?

Reply:
yeap! let me remove this from here and merge with the existing one 👍